### PR TITLE
fix(actions): Quick fix for actions not having any properties or steps

### LINKED ISSUE #876: NPE when an action has no properties
The error is:
```
vendor.1afd272e7651c79ca9cb.bundle.js:148 ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'properties' of undefined
TypeError: Cannot read property 'properties' of undefined
at e.project (1.444750b3405e7e6e3f99.chunk.js:1)
at e._next (vendor.1afd272e7651c79ca9cb.bundle.js:848)
at e.next (
```

### PR DESCRIPTION
**Pull request by zregvart**
Just a quick patch to prevent the NPE.

Fixes #876

### CODE REVIEW

File: src/app/integrations/edit-page/action-configure/action-configure.component.ts [hunk 1]
```diff
@@ -83,11 +83,15 @@ export class IntegrationsConfigureActionComponent extends FlowPage
...
-            ),
-          );
+          if (this.action.definition.propertyDefinitionSteps && this.action.definition.propertyDefinitionSteps.length > 0) {
+            this.formConfig = JSON.parse(
+              JSON.stringify(
+                this.action.definition.propertyDefinitionSteps[0].properties,
+              ),
+            );
+          } else {
+            formConfig = {};
```
Comment (kahboom) [inline src/app/integrations/edit-page/action-configure/action-configure.component.ts]: Could probably have declared this at the top, but either way it's good. 👍

**Comment by syndesis-dev** [comment]
ooops :)

**Comment by zregvart** [comment]
Thanks!

**Comment by kahboom** [comment]
thank _you_!

**Pull request merged**